sql/alps_submitter.go

```diff
@@ -6,18 +6,28 @@ import (
 	"strconv"
 )
 
-type GraphConfig struct {
+const (
+	SPARSE  = "SPARSE"
```

Comment:
In Go, we encourage using snake_case. ([mixed caps](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps)).

Reply:
Fixed